### DISCUSSION TITLE
Does something affect screen readers from reading clr-tooltip-content?

### DISCUSSION THREAD

**Discussion by ChrysaLa** [vmware-clarity/ng-clarity#798] [Q&A] [answered]
Hello!
I want to make screen readers read the content of a tooltip but i can't . I tried the cmd+option+control+slash but doesn't do anything. My current implementation is ` <clr-tooltip>
          <clr-icon
            clrTooltipTrigger
            shape="info-circle"
            class="info-circle"
            size="24"
            role="button"
            aria-label="info"
          ></clr-icon>
          <clr-tooltip-content [@tooltipAnimation] [clrPosition]="'top-right'" clrSize="md" *clrIfOpen>
            <span [@tooltipAnimation]
              >Tooltip text</span
            >
          </clr-tooltip-content>
        </clr-tooltip>` 
        Can someone check it and tell me if i am doing something wrong? 
        Thanks in advance!

**Comment by kevinbuhmann** [comment, marked as answer]
It works on https://clarity.design/documentation/tooltips. I am not sure why it isn't working with your code, but note that `clr-icon` is deprecated in favor of `cds-icon`. It is possible that you are using an unsupported version of `@clr/angular`. Please check the support policy: https://clarity.design/documentation/support.
